Azure-Samples/modern-data-warehouse-dataops
# ydaponte/604 include whl file

# Type of PR
- Documentation changes
- Code changes

## Purpose
- Adding wheel package: source code, .whl file, changes to upload package to workspace and spark pool.
- Altering notebooks to import whl package 
- Updating the ReadMe file

## Does this introduce a breaking change? If yes, details on what can break
No.

## Author pre-publish checklist


## Validation steps
- Deploy sample
- Run Main_0 pipeline
- Run Data Retention pipeline
- Check on the Monitoring tab that pipelines execute successfully

## Issues Closed or Referenced
- Closes #604
